docs/tutorials/PipelinesApi101.md

```diff
@@ -27,13 +27,14 @@ You may create a service account using the `gcloud` command, consider running th
 
 ```
 #!/bin/bash
+export LC_ALL=C 
```

Comment:
Is this necessary here or just good practice? 

FWIW I never saw this before and thought you'd somehow made a bad copy/paste until I googled it...

Reply:
I needed that in order to get urandom to output letters instead of utf-8 stuff that didn't work. I got it from Slack Overflow.